New structure/Backend/requirements.txt

```diff
@@ -0,0 +1 @@
+Django==2.2
```

Comment:
Why are we using such an old version of Django?

Reply:
I will change it to 3.2.
I was willing to write 3.2 maybe I made a typing error so sorry for that.